shotover/src/tls.rs

```diff
@@ -174,12 +177,13 @@ impl TlsConnector {
             })
             .transpose()?;
 
-        let config_builder = rustls::ClientConfig::builder().with_safe_defaults();
+        let config_builder = ClientConfig::builder();
         let config = match (private_key, certs, tls_config.verify_hostname) {
             (Some(private_key), Some(certs), true) => config_builder
                 .with_root_certificates(root_cert_store)
                 .with_client_auth_cert(certs, private_key)?,
             (Some(private_key), Some(certs), false) => config_builder
+                .dangerous()
```

Comment:
I too, like to live dangerously!

Reply:
Tbh I'd rather not 😆 
It's a direct port of what we were doing with openssl which gives no such warnings.
Was convinced to keep the functionality by a coworker.